getting-started/README.md

```diff
@@ -92,6 +131,32 @@ Run `docker ps -a` as an additional check to verify that all the services are ru
 
 ### Interact with ChatQnA
 
+
+::::{tab-set}
+
+:sync-group: Cloud Service Providers
+
+:::{tab-item} AWS
+
+:sync: AWS
+
+You can interact with ChatQnA via a browser interface: 
+
+* To view the ChatQnA interface, open a browser and navigate to the UI by inserting your externally facing IP address in the following: `http://{public_ip}:80’. 
+>**Note**: If you are having issue accessing the UI, you need to add an inbound rule in the security group. 
+
+* Look up Security Groups in the search bar and select the security group used when creating the instance. 
+* Click on the Edit inbound rules on the right side of the window. 
+* Select Add rule at the bottom, and create a rule with type as Custom TCP , port range as 80 and source as 0.0.0.0/0 

```

Comment:
you probably will be removed automatically by corporate security group if you open port 80 to all.....   65535 port for all case.   you might just mention put " select a port safe to open to public. ex: 65535"

Reply:
also, you might need to mention how to change NGINX port accordingly via environment variable.

Reply:
please also try on external IP under your security group. no worries if it also works for external IP.